### PR TITLE
added Tools component

### PR DESCRIPTION
**Pull request by Sakamoto448**
* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Added Tools Component to the application.

* **What is the current behavior?* What does the code do before you made the new changes?** 
There was no existing text other than Hello World

* **What is the new behavior?**
You may now see, what a list of tools the team will be using for the project.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
Everything seems to be in place and nothing is breaking, however the list is quite bland and will still need some work aesthetically.

* **Other information**:
UPDATE: Added a Tools.css to adjust the list with a logo image with the name of the tool.

### CODE REVIEW

File: src/components/Tools.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Tools() 
+{
+  return (
+      <div>
+        <h1> Jira </h1>
```
Comment (Codubee) [inline src/components/Tools.js]: Typically h1 tags are reserved for titles of pages. What do you think about using an [html list ](https://www.w3schools.com/html/html_lists.asp)or h2,h3,h4, or h6 tag instead?

Also, you may have asked a question about this before but can we add a title on top of this explaining what the section is for.

**Review by Codubee — CHANGES_REQUESTED** [review]
Left a comment for discussion

File: src/components/Tools.js (hunk 1 above)
Comment (BrandonAlexanderLopez) [reply src/components/Tools.js]: I agree with Miguel, shall I review after these changes?

**Review by Miguel-S117 — COMMENTED** [review]
Looks great and works! What do you think about moving the ExplanationPage feature above the Tools feature so the ExplanationPage will be shown at the top of the site and the list of tools below it?

**Review by BrandonAlexanderLopez — APPROVED** [review]

**Review by BrandonAlexanderLopez — DISMISSED** [review]

**Review by Miguel-S117 — DISMISSED** [review]

**Review by BrandonAlexanderLopez — DISMISSED** [review]

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**